generators/typescript/sdk/generator/src/contexts/type-schema/TypeSchemaContextImpl.ts

```diff
@@ -154,8 +164,12 @@ export class TypeSchemaContextImpl implements TypeSchemaContext {
         });
     }
 
+    private generateForInlineUnion(typeName: DeclaredTypeName): ts.TypeNode {
+        throw new Error("Inline unions are not supported in schemas.");
```

Comment:
These errors would only ever occur from a generator author issue, right? It will be impossible for some customer to hit this on their own? If so, let's preface this with `Internal error` like so:

```suggestion
        throw new Error("Internal error; inline unions are not supported in schemas.");
```

This might be relevant for other error messages, too.